client_secret.json

```diff
@@ -0,0 +1 @@
+{"installed":{"client_id":"561508962971-vem6e8jhh5dgk7e5fvbquuspb0bfcq4p.apps.googleusercontent.com","project_id":"vocal-etching-196113","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://accounts.google.com/o/oauth2/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"qkqwwhzX0WbQj5Q8Vob_ADVK","redirect_uris":["urn:ietf:wg:oauth:2.0:oob","http://localhost"]}}
```

Comment:
@thibautcornolti: are you sure you wanted to push this?

Reply:
@lfrancois thanks, it was a (big) mistake.